docs/tutorials/quickstart.md

```diff
@@ -22,31 +22,36 @@ To install Node.js, click [here](https://nodejs.org/en/download/).
 
 OASGraph can be used either as a library, or via its Command Line Interface (CLI). To install OASGraph, clone the repository and link the library (for the CLI commands to work) using the indicated steps.
 
-```
+```sh
+
 git clone git@github.com:strongloop/oasgraph.git
 cd oasgraph
 npm link
+
 ```
 
-### Save OAS
+### Locate the OAS
```

Comment:
I think we can clarify this by saying the user can either save or locate the OAS. I think mentioning one and not the other may be confusing.

Reply:
### Save or locate the OAS

OASGraph relies on the OpenAPI Specification (OAS) of an existing API to create a GraphQL interface around that API. OASGraph can also retrieve a web-hosted OAS. 

If you are using LoopBack, you can simply copy the URL location of the web-hosted OAS, which is usually [http://127.0.0.1:3000/openapi.json](http://127.0.0.1:3000/openapi.json).

***

If you want to generate a GraphQL interface for another API, make sure that API is running and proceed in the same way or save the OAS.